components/theme/colors.tsx

```diff
@@ -1,8 +1,18 @@
-export type ColorTypes = 'white' | 'black' | 'darkgrey' | 'lightgrey'
+import styles from '../../scss/__variables.module.scss'
+
+export type ColorTypes =
```

Comment:
Since this is going to be used by other people, we should turn this into a typescript enum to export, instead of type

Reply:
Oh actually I just looked at the codebase and I see `ColorTypes` is just used as purely a type.. so ignore my comment. This is fine